packages/image_picker/image_picker/example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -296,6 +349,13 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 		};
+		6801C8342555D726009DAF8D /* Resources */ = {
```

Comment:
This is the default Resources build phase that Xcode adds for every new target, like the Frameworks one above. This one is also empty, so it could be deleted, but most people keep the default ones.